src/common/catalog/categories/kubernetes-cluster.injectable.ts

```diff
@@ -4,12 +4,10 @@
  */
 import { getInjectable } from "@ogre-tools/injectable";
 import { KubernetesClusterCategory } from "../../catalog-entities/kubernetes-cluster";
-import { builtInCategoryInjectionToken } from "../category-registry.injectable";
 
 const kubernetesClusterCategoryInjectable = getInjectable({
   id: "kubernetes-cluster-category",
   instantiate: () => new KubernetesClusterCategory(),
-  injectionToken: builtInCategoryInjectionToken,
 });
```

Comment:
I don't think that this should be left in if this injection token is being removed. Because we expose this to the extension API an extension might try and listen for the events on this instance, but if it is not registered in the category registry then no events will be emitted on this instance.

Reply:
@Nokel81 what's your suggestion? We cannot expose main or renderer because the extension api is common.

Reply:
Well this will cause bugs so we cannot leave it like this.

Either we: 

1. Have only `kubernetesClusterCategoryInjectable` which then does `di.inject(kubernetesClusterCategoryColumnsInjectable)`
2. Have the extension API expose a `legacyGlobalForExtensionApi` based on a injection token and use a jump pad for the auto registration.

Reply:
@Nokel81 Yes I understand that we need to fix this. Still I started to wonder why are we exposing instance of `KubernetesClusterCategory`, shouldn't all instances be available through `CatalogCategoryRegistry`.... feels like a bug to me.